src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -772,8 +786,22 @@ public static async Task<string> ExecuteScriptFunctionAsync(WebView2 webView2, s
         /// <param name="type"></param>
         internal void UpdateContext(string type)
         {
-            ExecuteScriptFunctionAsync(browser,"libController.setHostContext", type);
-            ExecuteScriptFunctionAsync(browser, "replaceImages");
+            try
+            {
+                ExecuteScriptFunctionAsync(browser, "libController.setHostContext", type);
```

Comment:
I don't think this will work for a non-awaited async method call.

Reply:
oh yeah, that makes sense